package.json

```diff
@@ -111,6 +110,8 @@
     "node-mandrill": "1.0.1",
     "passport": "0.3.2",
     "passport-oauth2": "craigspaeth/passport-oauth2",
+    "pino": "^7.11.0",
```

Comment:
This is a dependency of `pino-http`. Do we also need to depend on it directly? (I guess this depends on the `timestamp` question.)